src/ucp/core/ucp_worker.c

```diff
@@ -642,7 +642,9 @@ ucs_status_t ucp_worker_arm(ucp_worker_h worker)
         wakeup = worker->wakeup.iface_wakeups[tl_id];
         if (wakeup != NULL) {
             status = uct_wakeup_efd_arm(wakeup);
-            if (status != UCS_OK) {
+            if (UCS_ERR_BUSY == status) {
+                result_status = status;
```

Comment:
if we got BUSY should return immediately - no point of arming others. it's enough that one tl could not be armed to fail the whole operation.